math/find_number_of_iteration_to_kaprekar_constant.cpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * @author [jasonchee816](https://github.com/jasonchee816)
+ * @file
+ *
+ * @brief [Program to count iterations needed to reach Kaprekar's Constant, 6174
+ * via Kaprekar's Routine](https://en.wikipedia.org/wiki/Kaprekar%27s_routine)
+ */
+
+#include <algorithm>
+#include <cassert>   /// for assert
+#include <iostream>  /// for IO operations
+#include <string>    /// for string
+
+/**

```

Comment:
```suggestion
/**
 * @namespace math
 * @brief Mathematical algorithms
 */
namespace math {
/**
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable